chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -949,6 +949,98 @@ Example
 - Check WMTS with Swiss ech-0056 profile (html): `https://api3.geo.admin.ch/owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS <../../../owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS>`_
 
 
+.. _terrain_service_description:
+
+---------------
+
+Terrain Service
+---------------
+
+A RESTFul implementation of "`Cesium <http://cesiumjs.org/>`_" `Quantized Mesh <https://cesiumjs.org/data-and-assets/terrain/formats/quantized-mesh-1.0.html>`_ terrain service.
+Terrain tiles are served according to the `Tile Map Service (TMS) <http://wiki.osgeo.org/wiki/Tile_Map_Service_Specification>`_ layout and global-geodetic profile.
+In order to access to the terrain tiles, you require a `swisstopo web access - WMTS documentation <http://www.swisstopo.admin.ch/internet/swisstopo/en/home/products/services/web_services/webaccess.html>`_.
+
+URL
+***
+
+- http://terrain0.geo.admin.ch or https://terrain0.geo.admin.ch
+- http://terrain1.geo.admin.ch or https://terrain1.geo.admin.ch
+- http://terrain2.geo.admin.ch or https://terrain2.geo.admin.ch
+- http://terrain3.geo.admin.ch or https://terrain3.geo.admin.ch
+- http://terrain4.geo.admin.ch or https://terrain4.geo.admin.ch
+
+Metadata Service
+****************
+
+The `layer.json` file determines which terrain tiles are available.
+
+- http://3d.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d/default/20151231/4326/layer.json or https://3d.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d/default/20151231/4326/layer.json
+
+Parameters
+**********
+
+A request is in the form:
+
+::
+
+    <protocol>://<ServerName>/<ProtocoleVersion>/ch.swisstopo.terrain.3d/<Stylename>/<Time>/<TileMatrixSetId>/<Zoom>/<X>/<Y>.<FormatExtension>
+
+with the following parameters:
+
+===================    ==================================   ==========================================================================
+Parameter              Example                              Explanation
+===================    ==================================   ==========================================================================
+Protocol               http ou https                        The protocol type
```

Comment:
Officially, this part is not `protocol` but `scheme`. See https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml